packages/router/src/lib/helpers/prepare.ts

```diff
@@ -4,16 +4,18 @@ import {
   decodeAuctionBid as _decodeAuctionBid,
   calculateExchangeAmount,
   getRateFromPercentage,
-  calculateExchangeWad,
+  getAmountsOut,
 } from "@connext/nxtp-utils";
 import { BigNumber } from "ethers";
+import { PriceImpactTooHigh } from "../errors/auction";
 
 import { AmountInvalid } from "../errors/prepare";
 
 const ROUTER_FEE = "0.05"; // 0.05%
 const EXPIRY_DECREMENT = 3600 * 24;
 const ONE_DAY_IN_SECONDS = 3600 * 24;
 const ONE_WEEK_IN_SECONDS = 3600 * 24 * 7;
+const AMPLIFICATION = "85";
```

Comment:
Right now, it is a constant value. I also think it could be configurable. router/src/config.ts. is it ok to read this value from env file or config json just like other configs?

Reply:
yep!